src/WebJobs.Script.WebHost/Extensions/ExceptionExtensions.cs

```diff
@@ -23,7 +23,7 @@ public static (string exceptionType, string exceptionMessage, string exceptionDe
 
             string exceptionType = innerException.GetType().ToString();
             string exceptionMessage = Sanitizer.Sanitize(innerException.Message);
-            string exceptionDetails = Sanitizer.Sanitize(innerException.ToFormattedString());
+            string exceptionDetails = Sanitizer.Sanitize(exception.ToFormattedString());
```

Comment:
we have a columns 'InnerExceptionMessage` and `InnerExceptionType` - we can use these for inner exceptions and leave details for outer exception?

Reply:
exception.ToFormattedString() actually logs all inner exception messages and types as well as part of formatting it with ---> separator (the screenshots above show this). Also a little unsure of doing that under the scope of this PR without knowing what existing detectors we might break (this change probably breaks a few, I don't know).

Reply:
what you have here looks right. `exceptionDetails` is supposed to capture full stack and for inner exception we have other columns.